connectors/src/pages/index.module.css

```diff
@@ -0,0 +1,149 @@
+.main {
```

Comment:
We might need it in the future, so should we keep it as an example of how to implement it or just remove it and figure out later?